include/cvc5/cvc5_skolem_id.h

```diff
@@ -732,6 +732,18 @@ enum ENUM(SkolemId)
    * - Sort: ``(-> FP Real)``
    */
   EVALUE(FP_TO_REAL),
+
+  /**
+   * A skolem function introduced by the int-blaster.
+   * Given a function f with argument and/or return types
+   * that include bit-vectors, we get a function
+   * that replaces them by integer types.
+   * For example, if the original function is from
+   * BV and Strings to Strings, the resulting
+   * function is from Ints and Strings to Strings.
+   */
```

Comment:
Include the same standard documentation about the number of skolem indices and return sort here (see examples above).